webpack/ForemanInventoryUpload/Components/InventorySettings/AdvancedSetting/AdvancedSettingsConstants.js

```diff
@@ -0,0 +1,30 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+
+export const settingsDict = {
+  autoUploadEnabled: {
+    name: 'allow_auto_inventory_upload',
+    label: `${__('Auto upload')} ${__('(enabled by default)')}`,
```

Comment:
cc @ShimShtein, do we need the `enabled by default` addition?

Reply:
It is in the UI mocks

Reply:
honestly, I think we can omit this addition
as we don't specify other settings defaults

Reply:
removed